back/engines/commercial/analysis/spec/acceptance/inputs_spec.rb

```diff
@@ -0,0 +1,102 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'rspec_api_documentation/dsl'
+
+resource 'Inputs' do
+  explanation 'Inputs (inputs and survey responses) in the context of an analysis'
+
+  before { header 'Content-Type', 'application/json' }
+
+  get 'web_api/v1/analyses/:id/inputs' do
+    with_options scope: :page, required: false do
+      parameter :number, 'Page number (starts at 1)'
+      parameter :size, 'Number of inputs per page'
+    end
+
+    with_options required: false do
+      parameter :search, 'Filter by searching in title and body'
```

Comment:
Is this true? What about the inputs from surveys where the information is stored in custom fields. Are they searchable?